src/pages/conference/speakers/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import styled from 'styled-components'
+import ImageContentBlock from './_speakers_list'
+import Name1 from './images/name1.svg'
+import Name2 from './images/name2.svg'
+import Name3 from './images/name3.svg'
+import Name4 from './images/name4.svg'
+import Footer from 'components/layout/footer'
+import { Localize } from 'components/localization'
+
+const Layout = styled.div`
+    background: var(--color-white);
+`
+
+const data = [
+    {
+        id: 'name_01',
+        image: Name1,
+        title: <Localize translate_text="Name" />,
+        content: (
+            <Localize translate_text="Who is the speaker? This is where you can write a bio to detail your speaker’s credentials and the unique perspective they will give to the conference." />
+        ),
+    },
+    {
+        id: 'name_02',
+        image: Name2,
+        title: <Localize translate_text="Name" />,
+        content: (
+            <Localize translate_text="Who is the speaker? This is where you can write a bio to detail your speaker’s credentials and the unique perspective they will give to the conference." />
+        ),
+    },
+    {
+        id: 'name_03',
+        image: Name3,
+        title: <Localize translate_text="Name" />,
+        content: (
+            <Localize translate_text="Who is the speaker? This is where you can write a bio to detail your speaker’s credentials and the unique perspective they will give to the conference." />
+        ),
+    },
+    {
+        id: 'name_04',
+        image: Name4,
+        title: <Localize translate_text="Name" />,
+        content: (
+            <Localize translate_text="Who is the speaker? This is where you can write a bio to detail your speaker’s credentials and the unique perspective they will give to the conference." />
+        ),
+    },
+]
+
+const Speakers = () => {
+    return (
+        <Layout>
+            {data.map(({ image, title, content, id }) => {
```

Comment:
```suggestion
            {data.map(({ image, title, content, id }) => {
               return <ImageContentBlock title={localize(title)} image={image} content={localize(content)} key={id} /> 
            }}
```
you can go further and update the `ImageContentBlock` component, so it can get the `TString` types and use the `localize` or `Localize` inside of it. it's gonna be even better, thanks :pray: